pkg/util/child.go

```diff
@@ -0,0 +1,30 @@
+// +build !linux
+
+/*
+Derived from:
+
+https://github.com/kubernetes/kubernetes
+
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+)
+
+// childSignals are the allowed signals that can be sent to children in Windows to terminate
+var childSignals = []os.Signal{os.Interrupt, os.Kill}
```

Comment:
(Amending my previous comment)

> on Windows sigkill is an equivalent of window close afaik, which is not the same as kill on UNIX.

Is that an argument to handle SIGKILL on Windows AND Darwin, and not on Linux?

As the docs say, we cannot catch SIGKILL using `signal.Notify`. Short demo: https://gist.github.com/rhcarvalho/71bea7d3dd2b969bfac8e244f9810963

So, @bparees, addressing your concern, it makes no sense to "try to catch" SIGKILL.